dsc_lib/src/dscresources/command_resource.rs

```diff
@@ -614,6 +614,11 @@ pub fn invoke_command(executable: &str, args: Option<Vec<String>>, input: Option
     };
 
     if exit_code != 0 {
+        if let Some(exit_codes) = exit_codes {
+            if let Some(error_message) = exit_codes.get(&exit_code) {
+                return Err(DscError::Command(executable.to_string(), exit_code, error_message.to_string()));
```

Comment:
Should have a new error type for this in case we want to change the message or make it more clear it's mapped